stripes-web/src/main/java/org/stripesframework/web/controller/multipart/DefaultMultipartWrapperFactory.java

```diff
@@ -111,6 +111,10 @@ public void init( Configuration config ) throws Exception {
       // See if a maximum post size was configured
       String limit = config.getBootstrapPropertyResolver().getProperty(MAX_POST);
       if ( limit != null ) {
+         if ( "unlimited".equals(limit) ) {
+            _maxPostSizeInBytes = Long.MAX_VALUE;
+         }
+
```

Comment:
I don't see much value in this special case. It can be manually set to `Long.MAX_VALUE` in the configuration or better some reasonable limit. Having multiple layers of checks is usually a good idea.